docs/DeveloperGuide.md

```diff
@@ -922,6 +922,34 @@ Currently, the process of modifying patient tags in the PatientSync application
 To address this, we are planning to introduce the EditTagsCommand feature, providing users with a more flexible and efficient way to manage patient tags. The planned EditTagsCommand feature is designed to enhance user productivity, reduce the likelihood of errors, and improve overall usability within PatientSync.
 
 
+### 4.3 Input Validation for Events
+
+Presently, the Date and Datetime for Events, referred to as `DATE_OR_DATETIME_OF_EVENT`, do not have sufficient input validation. For example, the user is currently able to input `30-02-2024, 24:00 - 24:00`. However, `30-02-2024` is not a valid Date and `24:00` is not a valid time.
```

Comment:
Just to check: are the current input validation implemented already mentioned in the Events section of the DG? if not, I suggest could include the current validation checks performed!

Reply:
not validated properly lmao